### DISCUSSION TITLE
Mod for accuracy weight adjustment

### DISCUSSION THREAD

**Discussion by WhiteNight1357** [ppy/osu#28428] [Ideas] [answered]
I have always wanted to change weights of judgement in calculating accuracy. For example, I wanted to change osu!maina accuracy calculation to scorev1 style, or style of other games like chunithm or iidx. So I made mod for it, but I'm not sure whether other players would want it. 
Demonstration of the mod is [Here](https://youtu.be/K5ycH4ikQxE?si=-DGvyUvppFuNeHYg)
Note: this mod does not affect result accuracy, only the accuracy displayed during gameplay

**Comment by bdach** [comment, marked as answer]
"other players wanting it" is not a factor here. This is not happening "because it's possible" because this impacts many other things such as:

- how to display scores with different notions of "accuracy" next to each other on leaderboards
- what such weighting changes mean for granting medals
- how this accuracy weighting impacts scoring (since score is derived from accuracy)
- etc.

and therefore introduces untenable levels of complexity to the game.

If you have an actual problem with the judgement weightings as they are in the game right now for whatever reason, then we can discuss the underlying reason for that and consider changing the actual weightings to resolve the problems. But this is *not* going to be configurable even if there is "player demand" for it.

**Reply by WhiteNight1357** [reply]
I think you have misunderstood my point.

My point is changing accuracy weight for accuracy meter shown during gameplay, not the accuracy used for scoring.
This would affect neither leaderboard, medals, nor scoring. I know using different weights for displayed accuracy and accuracy of score might induce confusion, but score and accuracy already has little correlation because of combo portion of scorev2. Also, because it is mod, people who watch replay, spectate, or watch play video on youtube would know that the play used adjusted accuracy weight for display.

I think purpose of accuracy display is letting know the player how good they are playing right now. Some players focus on getting better perfect:great ratio, others focus on full combo, and others focus on general accuracy. By letting them adjust weights of judgements, they could have better indicator of how good they are playing from the perspective of what they pursue.
